### PR TITLE
VIH-7950 Update mapper for interpreter-individual linked participants in hearings

### PR DESCRIPTION
**Pull request by alberto-hortiguela**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7950

### Change description ###
Participant mapper update to set order on linked participants

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts [hunk 1]
```diff
@@ -59,6 +60,9 @@ export class ParticipantPanelModelMapper {
         });
         const allMapped = linked.map(l => new ParticipantPanelModel(l));
         allMapped.push(new ParticipantPanelModel(participant));
-        return allMapped;
+        return [
```
Comment (AWildBugAppeared) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts:63]: Can you add tests for this?

**Review by AWildBugAppeared — CHANGES_REQUESTED** [review]

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { ParticipantForUserResponse, Role, RoomSummaryResponse } from 'src/app/s
 import { LinkedParticipantPanelModel } from 'src/app/waiting-space/models/linked-participant-panel-model';
 import { PanelModel } from 'src/app/waiting-space/models/panel-model-base';
 import { ParticipantPanelModel } from 'src/app/waiting-space/models/participant-panel-model';
+import { HearingRole } from 'src/app/waiting-space/models/hearing-role-model';
 
 export class ParticipantPanelModelMapper {
     mapFromParticipantUserResponse(pats: ParticipantForUserResponse[]): PanelModel[] {
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts:8]: is the "pats" argument not actually used in this function? Or am I reading it worng

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts (hunk 1 above)
Comment (alberto-hortiguela) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts:63]: As far as I am aware we cannot test private methods so this mapper would have to go public/protected

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts (hunk 2 above)
Comment (alberto-hortiguela) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts:8]: the **pats** argument is used to return a new var based on the mapping result scenarios

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts (hunk 1 above)
Comment (AWildBugAppeared) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts:63]: I'll give you a call.

Comment (alberto-hortiguela) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts:63]: unit test added

**Review by AWildBugAppeared — APPROVED** [review]

**Pull request merged**